jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -571,6 +571,23 @@ public ClassLoader getClassLoader() {
 	return classLoader;
 }
 
+/**
+ * Returns the classloader used to load the receiver's class.
+ * Returns null if the class was loaded by the bootstrap (system) class loader.
+ * This skips security checks.
+ *
+ * @return		the receiver's class loader or null
+ *
+ * @see			java.lang.ClassLoader
+ */
+ClassLoader internalGetClassLoader() {
+	ClassLoader result = classLoader;
+	if ((null != classLoader) && (classLoader == ClassLoader.bootstrapClassLoader))	{
```

Comment:
The null check here is unnecessary.  Checking for `(classLoader == ClassLoader.bootstrapClassLoader)` is sufficient.

Reply:
Okay.